microsoft/PowerToys
# [ManagedCommon.dll & ManagedTelemetry.dll] Unify exe/dll naming

## Summary of the Pull Request

**What is this about:**
Rename `ManagedCommon.dll` to `PowerToys.ManagedCommon.dll`
Rename `ManagedTelemetry.dll` to `PowerToys.ManagedTelemetry.dll`

**What is include in the PR:** 

**How does someone test / validate:** 
 - start PT
 - check if some of the modules that are using ManagedCommon.dll and ManagedTelemetry.dll works (e.g. FZ Editor)

## Quality Checklist

- [x] **Linked issue:** #2125

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
